CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to Open Liberty Guides
+
+Anyone can contribute to the Open Liberty guides and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc. 
+
+## Raising Issues
+
+Please raise any bug reports on the Open Liberty guide's repository: https://github.com/OpenLiberty/[guide-name]/issues, where `[guide-name]` is the repository name of the guide . For new topics, large updates to existing guides, or general suggestions and ideas, report your issue on the [Open Liberty common guides repository](https://github.com/OpenLiberty/guides-common/issues). Be sure to search the list to see if your issue has already been raised.
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributions 
+
+Contributing to the Open Liberty guides is simple. All you need to do is create your own branch to include your contribution(s). If you want to make updates to existing guides, create a new branch and open a pull request to the `qa` (quality assurance) branch.
+
+### Text or Content Contributions
+
+If you are contributing text of content changes via a pull request and you certify the [Developer Certificate of Origin (DCO)](https://developercertificate.org)), add a line to the end of the Git commit message to sign your work:
+
+```
+Signed-off-by: Jane Williams <jane.williams@gmail.com>
+```
+
+The sign-off is just a line at the end of the commit message that certifies that you wrote it or otherwise have the right to pass it on as an open source patch.
+
+Use your real name when you sign. We can't accept pseudonyms or anonymous contributions. To open a pull request to the `qa` branch, follow the official [git documentation](https://git-scm.com/docs/git-request-pull). If reviewers agree with your change, your change will be merged to the `master` branch for publishing.
```

Comment:
- [ ] I don't think need the git documentation and I doubt the example has ` v1.0` will work for us

Reply:
This was from when you suggested to add instructions on cloning, creating a PR, etc... I removed nonetheless.